### DISCUSSION TITLE
can please tell why my PR labeled as Invalid

### DISCUSSION THREAD

**Discussion by damarudhvarma** [Kushal997-das/Project-Guidance#1434] [General]
@Kushal997-das  can you  tell the reason why  my PR "AI Image Generator #1388 " is marked invalid ,I've  worked very hard to build it

**Comment by Kushal997-das** [comment]
> @Kushal997-das can you tell the reason why my PR "AI Image Generator #1388 " is marked invalid ,I've worked very hard to build it

Click that label that check pop up .

**Reply by damarudhvarma** [reply]
oh thanks, I got It I'm a beginner to open-source contributions.I raised issue #1435  please assign it to me
